README.md

```diff
@@ -165,11 +164,6 @@ this:
 `\section{Final Aftermath:}`
 
 
-### Disclaimers and Epigraphs
```

Comment:
I think we should say that some have been omitted but are in the source files.

Reply:
done